### PR TITLE
Version simplify

### PR DESCRIPTION
**Pull request by jburel**
Read version from package.json
Only one file will need to be updated when the version changes

cc @will-moore @sbesson

### CODE REVIEW

**Comment by sbesson** [comment]
With this PR included https://ci.openmicroscopy.org/view/Failing/job/WEB-DEV-trial-deploy/319/ (and WEB-DEV-merge-deploy) failed with

```
[Errno 2] No such file or directory: '/opt/hudson/workspace/WEB-DEV-trial-deploy/trial-venv/lib/python2.7/site-packages/omero_figure/../package.json'
```

The single-source version file might have located to be within the package root directory.

**Comment by jburel** [comment]
Just noticed that
excluding for now

--exclude

File: omero_figure/views.py [hunk 1]
```diff
@@ -109,16 +109,14 @@ def index(request, file_id=None, conn=None, **kwargs):
...
-
     script_service = conn.getScriptService()
     sid = script_service.getScriptID(SCRIPT_PATH)
     script_missing = sid <= 0
     user_full_name = conn.getUser().getFullName()
 
     context = {'scriptMissing': script_missing,
                'userFullName': user_full_name,
-               'version': version}
+               'version': utils.__version__}
```
Comment (will-moore) [inline omero_figure/views.py]: I think you want ```utils.get_version()``` here, since in the UI I'm seeing ```0.0.0```.

**Comment by jburel** [comment]
--exclude

**Comment by jburel** [comment]
@will-moore did you have a chance to look at it?

File: omero_figure/utils.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    p = os.path.abspath(fname)
+    with open(p) as f:
+        if content_type in ('json',):
+            data = json.load(f)
+        else:
+            data = f.read()
+    return data
+
+
+def read_version(file_name):
```
Comment (will-moore) [inline omero_figure/utils.py]: Doesn't look like you're using this function anywhere now. Do you still want to keep it?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "omero-figure",
-  "version": "3.0.0",
+  "version": "0.0.0",
```
Comment (will-moore) [inline package.json]: A comment here that this is a placeholder and the real version in in utils.py might help avoid confusion.

**Comment by will-moore** [comment]
A couple of minor comments, but the functionality is working fine (UI shows correct version).

File: omero_figure/utils.py (hunk 2 above)
Comment (jburel) [inline omero_figure/utils.py]: probably no need to keep it

**Comment by jburel** [comment]
@will-moore: done

File: package.json (hunk 3 above)
Comment (will-moore) [inline package.json]: Looks good - I'd forgotten you can't just put comments directly in JSON.

**Comment by will-moore** [comment]
Looks good, and working fine in UI showing "3.0.0".

**Pull request merged**